src/Stack/SDist.hs

```diff
@@ -24,7 +24,7 @@ import qualified Data.ByteString as S
 import qualified Data.ByteString.Char8 as S8
 import qualified Data.ByteString.Lazy as L
 import           Data.Char (toLower)
-import           Data.Data (cast)
+import           Data.Data (cast, gmapT)
```

Comment:
This is also fixed in the 0.1.9.1 bump to rio.

Reply:
Sure, but that release came a bit later in time :)

Reply:
Yes, you're right. It just wasn't immediately clear that we were going to add these back in 0.1.9.1, so I wanted to mention it.